staff/prisma/seed.ts

```diff
@@ -0,0 +1,34 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+// Any interactions with Prisma will be async
+// eslint-disable-next-line @typescript-eslint/require-await
+async function seed() {
+  // Put the actions you need to take to seed the databse here.
+  // You can access relations as normal here
+  // const users = await prisma.user.findMany(); // (for example)
+  await prisma.localAgency.createMany({
+    data: [
+      {
+        urlId: "gallatin",
+        name: "Gallatin WIC",
+      },
+      {
+        urlId: "missoula",
+        name: "Missoula WIC",
+      },
+    ],
+  });
+  console.log(`Database has been seeded. 🌱`);
+}
+
+seed()
+  .then(async () => {
+    await prisma.$disconnect();
+  })
+  .catch(async (e) => {
+    console.error(e);
+    await prisma.$disconnect();
```

Comment:
is this commit pollution in the diff? 🤔

Reply:
I can drop this from this PR. I included it because I want to be able to seed the database the staff portal uses with some test data. But I didn't actually write up the seed data.